src/fulfiller_identity_client.js

```diff
@@ -106,8 +107,10 @@ class FulfillerIdentityClient {
       })
       .then(f => new Fulfiller(f.fulfillerId, f.internalFulfillerId, f.name, f.email, f.phone, f.language, f.links, f.archived))
       .catch((err) => Promise.reject(err.response && err.response.status === 404 ?
-        new FulfillerNotFoundError(`Fulfiller ${fulfillerId} does not exits`) :
-        new Error("Unable to get fulfiller: " + err.message)));
+        new FulfillerNotFoundError(`Fulfiller ${fulfillerId} does not exist`) :
+        err.response && err.response.status === 403 ?
```

Comment:
err.response is already checked in the first condition.

Reply:
True. However, I'd like to make all conditions for each case explicit: `err.response && err.response === 403` and `err.response && err.response === 404` occupy separate lines. This makes it easier to later add a condition which does not depend on `err.response`.

Alternatively, I could have ifs and else ifs.

Reply:
Changed to ifs and else ifs.